modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -1487,4 +1487,34 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
       }));
     });
   });
+
+  context('TINY-9600: sanitize: false', () => {
+    const testDisablingSanitization = (testCase: { name: string; input: string }) => {
+      it(testCase.name, () => {
+        const parser = DomParser({ sanitize: false }, schema);
+        const serializedHtml = serializer.serialize(parser.parse(testCase.input));
+
+        assert.equal(serializedHtml, testCase.input);
+      });
+    };
+
+    Arr.each([{
+      name: 'should not remove script tags',
+      input: '<script>alert(1)</script>'
+    }, {
+      name: 'should not remove iframe tags with child nodes',
+      input: '<iframe src="https://example.com"><p>Lorem ipsum</p></iframe>'
```

Comment:
Hmm, the Safari entity encoding branch case is not happening here. So is the editor somehow doing special filtering only for Safari. Or is dompurifier doing something here since it's disabled for this test.

Reply:
To be investigated in TINY-9624.